### PR TITLE
Stream and user banning

### PR DESCRIPTION
**Pull request by vaindil**
Ban the cucks.

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -144,3 +144,13 @@ export const toggleChat = isOtherChatActive => {
...
+
+export const STREAM_BANNED = Symbol('STREAM_BANNED');
+export const streamBanned = () => dispatch => {
+  //browserHistory.push('/');
+  alert('stream banned or something');
+  dispatch({
+    type: STREAM_BANNED,
+    payload: null,
+  });
+};
```
Comment (KenanY) [inline src/actions/index.js]: I don't think this addition is used anywhere, since the action was ultimately implemented in `src/actions/websocket.js`.

File: src/api/websocket.js [hunk 2]
```diff
@@ -214,6 +232,7 @@ export default function makeWebSocketServer(server) {
             'STREAMS_SET',
             streams,
           ]));
+
```
Comment (KenanY) [inline src/api/websocket.js]: Unnecessary whitespace addition.

**Comment by vaindil** [comment]
@KenanY Requested changes made.

**Comment by KenanY** [comment]
I fixed some linting errors and squashed everything into two commits (https://github.com/ILiedAboutCake/Rustla2/commit/8e793c0fff79169beb084d25c5e9914dbe97a048 and https://github.com/ILiedAboutCake/Rustla2/commit/ef352b0bf96fa9d99af069b3d1e6d1d52dace984)
